configurations.md

```diff
@@ -0,0 +1,1178 @@
+---
+---
+
+# User Configuration Files
+
+<div class="row">
+<div class="col-lg" markdown="1">
+
+## Developer Configurations
+
+Below are the configurations of a few of xmonad's maintainers.  Just
+keep in mind that these setups are very customized and perhaps a little
+bit hard to replicate (some may rely on features only available in
+personal forks or git), may or may not be documented, and most aren't
+very pretty either :)
+
+<div class="list-col-4" markdown="1">
+  - [byorgey](https://github.com/byorgey/dotfiles)
+  - [geekosaur](https://github.com/geekosaur/xmonad.hs/tree/pyanfar)
+  - [liskin](https://github.com/liskin/dotfiles/tree/home/.xmonad)
+  - [psibi](https://github.com/psibi/dotfiles/tree/master/xmonad)
+  - [slotThe](https://gitlab.com/slotThe/dotfiles/-/tree/master/xmonad/.config/xmonad)
+  - [TheMC47](https://github.com/TheMC47/dotfiles/tree/master/xmonad/.xmonad)
+</div>
+
+## Legacy Configurations
+
+These are currently part of the `xmonad-contrib` package but will be
+removed at some point in the future.  People importing them should copy
+the relevant functions into their own configurations.
+
+### XMonad.Config.Arossato
+
+<details>
+  <summary>xmonad.hs</summary>
+
+{% highlight haskell %}
+{% raw %}
+
+    {-# OPTIONS_GHC -fno-warn-missing-signatures #-}
```

Comment:
This content is double-indented, probably because `{% highlight haskell %}` starts a pre/code block and then it's indented again. I'm also not entirely sure whether the `{% raw %}` is needed or whether this affects anything.

Anyway, with `<details markdown=1>`, you can just use ` ```haskell ` like we do elsewhere in xmonad-web and other markdown content, so I'd probably recommend using that.

Reply:
Thanks!  Using the `haskell` highlight does indeed look cleaner: 7135df074d18103162cb3e970dc2e3b10954d17d

`{% raw %}` is needed[^1] because jekyll tries to somehow process the source while highlighting it with `Liquid` and some configs contain characters sequences that it's not happy with (I assume they are some kind of preprocessing directives for Liquid).  E.g., X.C.Dmwit with just a Haskell highlight results in

```
  Liquid Exception: Liquid syntax error (line 455): Variable '{{{ {-# LANGUAGE ExistentialQuantification, NoMonomorphismRestriction, TypeSynonymInstances, ViewPatterns, LambdaCase #-}' was not properly terminated with regexp: /\}\}/ in configurations.md
             Error: Liquid syntax error (line 455): Variable '{{{ {-# LANGUAGE ExistentialQuantification, NoMonomorphismRestriction, TypeSynonymInstances, ViewPatterns, LambdaCase #-}' was not properly terminated with regexp: /\}\}/
             Error: Run jekyll build --trace for more information.
```

[^1]: At least in some cases; I've removed all of the unecessary uses of it

Reply:
> `{% raw %}` is needed…

Oh, right, seems those `{{{` for vim (presumably) folding really do confuse Jekyll/Liquid even in ` ```haskell ` blocks. I don't know why I thought it might be related to something else. Feel free to autosquash and merge then. :-)